.env.private

```diff
@@ -1,7 +1,7 @@
 # feature flags
 REACT_APP_FEATURE_MIXPANEL=false
 REACT_APP_FEATURE_CHATWOOT=false
-REACT_APP_FEATURE_LIFI_SWAP=false
+REACT_APP_FEATURE_LIFI_SWAP=true
```

Comment:
```suggestion
```

Reply:
Might as well keep it ? This was a revert of https://github.com/shapeshift/web/pull/6524 which disabled it on prod envs only.